tests/acceptance/seller/InternalTransferCest.php

```diff
@@ -53,6 +58,33 @@ public function ensureTransferIsWorkingCorrectly(Seller $I, Example $example): v
         $this->index->seeTransferActionSuccess();
     }
 
+    private function checkIfUserHaveNegativeBalance(Seller $I, array $billInfo): array
+    {
+        $sum = $this->getTotalSumOnUserAccount($I, $billInfo);
+
+        $sum = $this->transformSum($sum);
+        if ($sum == null) {
```

Comment:
```suggestion
        if ($sum === null) {
```
завжди зрівнюй чітко